docs/config.md

```diff
@@ -197,6 +201,47 @@ names:
   commonNamePattern: "{host}.buoyant.io"
 ```
 
+#### Load Balancer
+
+* *loadBalancer* -- Specifies a load balancer to use.  It must be an object
+containing keys:
+  * *kind* -- One of the supported load balancers.
+  * Any options specific to the load balancer.
+
+Current load balancers include:
+
+[p2c]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-least-loaded
+[ewma]: https://twitter.github.io/finagle/guide/Clients.html#power-of-two-choices-p2c-peak-ewma
+[aperture]: https://twitter.github.io/finagle/guide/Clients.html#aperture-least-loaded
+[heap]: https://twitter.github.io/finagle/guide/Clients.html#heap-least-loaded
+
+##### [p2c][p2c]
+
+p2c supports the following options (see [here][p2c] for option semantics and defaults):
+
+* *maxEffort* -- Optional.
+
+##### [ewma][ewma]
+
+ewma supports the following options (see [here][ewma] for option semantics and defaults):
+
+* *maxEffort* -- Optional.
+* *decayTimeMs* -- Optional.
+
```

Comment:
That's right.  What we call ewma here is actually p2c+ewma.  Similarly, what we call aperture is actually aperture+least loaded.  The finagle docs (which we link to) spell this out more explicitly.